File: Server - C#/model/Flight.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FlightHubC_.Domain
{
    [Serializable]

    public class Flight : Entity<Guid>
    {
        public string Destination { get; set; }
        public DateTime DateTime { get; set; }
        public int AvailableSeats { get; set; }

        public string Airport;
        public Flight(string destination, DateTime date, int availableSeats, string airport)
        {
            Destination = destination;
            DateTime = date;
            AvailableSeats = availableSeats;
            Airport = airport;
        }
    }
}

File: Client - Java/Shared/src/main/java/org/flighthub/Domain/Flight.java
package org.flighthub.Domain;

import java.time.LocalDateTime;
import java.util.UUID;

public class Flight extends Entity<UUID> {
    private String destination;
    private LocalDateTime dateTime;
    private Integer availableSeats;
    private String airport;

    public Flight(String destination, LocalDateTime dateTime, Integer availableSeats, String airport) {
        this.destination = destination;
        this.dateTime = dateTime;
        this.availableSeats = availableSeats;
        this.airport = airport;
    }

    public String getDestination() {
        return destination;
    }

    public LocalDateTime getDateTime() {
        return dateTime;
    }

    public Integer getAvailableSeats() {
        return availableSeats;
    }

    public String getAirport() {
        return airport;
    }

    ;
}
